eclipse-collections/src/main/java/org/eclipse/collections/impl/set/sorted/mutable/MutableSortedSetFactoryImpl.java

```diff
@@ -25,6 +25,16 @@ public <T> MutableSortedSet<T> empty()
         return TreeSortedSet.newSet();
     }
 
+    @Override
+    public <T> MutableSortedSet<T> empty(Comparator<? super T> comparator)
+    {
+        if (comparator == null)
```

Comment:
Hi @Desislav-Petrov, I went and looked and was curious why we didn't have the similar null handling logic for Sets. I would prefer to keep the logic the same as it looks like TreeSet itself has the null handling logic.

Reply:
Got u - I was trying to be consistent with what we do in the other place but I guess for sets is different. Will fix.

Reply:
Thanks!

Reply:
hi @donraab - the null check for sortedset is removed now